org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/symbol/DocumentSymbolService.java

```diff
@@ -55,7 +56,7 @@
  * @author kosyakov - Initial contribution and API
  *
  * Contributors: 
- *   Rub�n Porras Campo (Avaloq Evolution AG) 
+ *   Rub�n Porras Campo (Avaloq Evolution AG) 
```

Comment:
encoding issue here

Reply:
I was wondering about that as well. Do you know which encoding should I use?